scenes/quests/lore_quests/quest_001/1_music_puzzle/components/trees.gd

```diff
@@ -3,13 +3,14 @@
 @tool
 extends Node2D
 
-const WOOL_PERCENTAGE: float = 0.4
+const ALTERNATIVE_PERCENTAGE: float = 0.4
 
-const TREE_WOOL_GREEN_01 = preload(
-	"res://scenes/game_elements/props/tree/components/Tree_Wool_Green_01.png"
-)
-const TREE = preload("res://scenes/game_elements/props/tree/components/Tree.png")
+const ALTERNATIVE_SPRITEFRAMES = preload("uid://djwymcffy83")
+const SPRITEFRAMES = preload("uid://d36eq8tqdaxdy")
 
 @export_tool_button("Randomize Trees Appearances") var a = func():
 	for child in get_children():
-		child.set("tree__texture", TREE if randf() > WOOL_PERCENTAGE else TREE_WOOL_GREEN_01)
+		child.set(
+			"sprite_frames",
+			SPRITEFRAMES if randf() > ALTERNATIVE_PERCENTAGE else ALTERNATIVE_SPRITEFRAMES
+		)
```

Comment:
For a different PR, but, I think it'd be really cool if this was configurable, maybe with an exported dictionary that goes from sprite frames to weight?

Reply:
I think the way the scenes are currently dressed is by changing trees one by one. I kept this randomizer because it was an easy change, otherwise I would have removed the script.